lib/WebpackOptionsApply.js

```diff
@@ -193,6 +194,12 @@ class WebpackOptionsApply extends OptionsApply {
 			ExternalsPlugin = require("./ExternalsPlugin");
 			compiler.apply(new ExternalsPlugin(options.output.libraryTarget, options.externals));
 		}
+
+		if(!options.output.devtoolNamespace) {
+			options.output.devtoolNamespace = options.output.library;
+		}
+		ModuleFilenameHelpers.setModuleNamespace(options.output.devtoolNamespace);
```

Comment:
Pass `namespace` to the SourceMapDevtoolPlugin (next section) instead. Similar to `devtoolModuleFilenameTemplate`